### PR TITLE
Don't allow too small bounding boxes because they can't be deleted manually after creation

### PR DESCRIPTION
**Pull request by yCobanoglu**

### CODE REVIEW

**Comment by yCobanoglu** [comment]
@jlaasonen it won't be possible to test this as it is impossible to create an annotation. Luis told me about this issue

**Comment by jlaasonen** [comment]
> @jlaasonen it won't be possible to test this as it is impossible to create an annotation. Luis told me about this issue

How do you know it's fixed if it is not possible to test?

Please use descriptive PR names and commit messages. What bug is this?

**Comment by yCobanoglu** [comment]
@jlaasonen tested it manually this PR stops from user making extremely small boxes which are impossible to delete

File: src/fragmentarium/ui/image-annotation/annotation-tool/FragmentAnnotation.tsx [hunk 1]
```diff
@@ -236,12 +236,15 @@ function FragmentAnnotation({
...
         setIsChangeExistingMode(false)
       } else if (geometry) {
-        const newAnnotation = new Annotation(geometry, {
-          ...data,
-          id: uuid4(),
-        })
-        setAnnotation({})
-        setAnnotations([...annotations, newAnnotation])
+        const minSize = Math.min(geometry.height, geometry.width)
+        if (minSize >= 0.3) {
```
Comment (jlaasonen) [inline src/fragmentarium/ui/image-annotation/annotation-tool/FragmentAnnotation.tsx:240]: Extract method with a proper name to clarify what is going on. Avoid magic numbers.

**Review by jlaasonen — CHANGES_REQUESTED** [review]
Add a proper title for the PR.

File: src/fragmentarium/ui/image-annotation/annotation-tool/FragmentAnnotation.tsx (hunk 1 above)
Comment (jlaasonen) [reply src/fragmentarium/ui/image-annotation/annotation-tool/FragmentAnnotation.tsx:240]: If you extract a method it also becomes testable.

File: src/fragmentarium/ui/image-annotation/annotation-tool/FragmentAnnotation.test.tsx [hunk 2]
```diff
@@ -177,3 +178,22 @@ it('delete everything', async () => {
...
+    width: 0,
+    type: 'RECTANGLE',
+  }
+  const geometryValid = {
+    x: 0,
+    y: 0,
+    height: 0.35,
+    width: 0.35,
+    type: 'RECTANGLE',
+  }
```
Comment (jlaasonen) [inline src/fragmentarium/ui/image-annotation/annotation-tool/FragmentAnnotation.test.tsx:196]: You can use [test.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout) when there are multiple data for the same test.

**Review by jlaasonen — APPROVED** [review]

**Pull request merged**